compatibility-verifier/checkoutAndBuild.sh

```diff
@@ -18,18 +18,23 @@
 # under the License.
 #
 
-
-# get a temporary directory in case the workingDir is not provided by user
-TMP_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
-cmdName=`baseName $0`
-source `dirname $0`/utils.inc
+cmdName=`basename $0`
+cmdDir=`dirname $0`
+source ${cmdDir}/utils.inc
 
 # get usage of the script
 function usage() {
-  echo "Usage: $cmdName -o olderCommit -n newerCommit [-w workingDir]"
+  echo "This command sets up the two builds to test compatibility in the working directory"
+  echo "Usage: $cmdName [-o olderCommit] [-n newerCommit] -w workingDir"
```

Comment:
Is it possible it print out the older and newer commits when the script is triggered?

Reply:
It is, but there is so much material printed, that it will get lost. The easiest way is for the user to just cd to the right directory under the working dir and get the commit versions/tags/ etc.